packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js

```diff
@@ -80,14 +127,67 @@ const NavigationPanel = ( { isOpen } ) => {
 				</div>
 
 				<div className="edit-site-navigation-panel__scroll-container">
-					{ contentActiveMenu === MENU_ROOT && (
-						<TemplatesNavigation />
-					) }
-					{ templatesActiveMenu === MENU_ROOT && (
-						<ContentNavigation
-							onActivateMenu={ setContentActiveMenu }
-						/>
-					) }
+					<Navigation
+						activeItem={ activeItem }
+						activeMenu={ activeMenu }
+						onActivateMenu={ ( menu ) => {
+							if ( menu === MENU_ROOT ) {
+								setNavigationPanelActiveMenu( menu );
+								setContentActiveMenu( menu );
+							} else if ( menu.startsWith( MENU_TEMPLATES ) ) {
```

Comment:
This was a goof. I didn't look closely enough at the titles of the template menus. They do not all start with "templates". For the next commit, I've flipped around the logic to check for "content" instead.